Tickets/dm 5503

r-owen opened the pull request.

PaulPrice commented on python/lsst/obs/sdss/converttsField.py:
```diff
@@ -21,14 +21,30 @@
+
+    @param[in] infile  path to tsField FITS file
+    @param[in] filt  index of filter in tsField FILTERS metadata entry
+    @param[in] exptime  exposure time (sec)
```
numpydoc?

PaulPrice commented on tests/testSdss.py:
```diff
@@ -71,11 +72,15 @@ def testGetDR7(self):
-            self.assertAlmostEqual(calib.getExptime(), 53.907456, 6)
-            self.assertAlmostEqual(gain, 4.72, 2)
+            tsField = ref.get("tsField")
+            print("tsField=", tsField)
```
`from future import print_function`?

r-owen commented:
Implemented all changes. I added from future import absolute_import, division, print_function to testSdss.py and removed the print statement because it was meant as a temporary debugging measure.
